### PR TITLE
release, interfaces: add new release.AppArmorFeatures helper

### PR DESCRIPTION
**Pull request by mvo5**
Followup from #4579

### CODE REVIEW

File: release/apparmor.go [hunk 1]
```diff
@@ -115,3 +116,19 @@ func probeAppArmor() (AppArmorLevelType, string) {
...
+	if err != nil {
+		return nil
+	}
+	appArmorFeatures := make([]string, 0, len(dentries))
+	for _, f := range dentries {
+		if isDirectory(filepath.Join(appArmorFeaturesSysPath, f.Name())) {
+			appArmorFeatures = append(appArmorFeatures, f.Name())
+		}
+	}
+	return appArmorFeatures
```
Comment (zyga) [inline release/apparmor.go]: Think we should sort them.

File: release/apparmor_test.go [hunk 2]
```diff
@@ -73,3 +73,14 @@ func (s *apparmorSuite) TestProbeAppArmorFullAppArmor(c *C) {
 	c.Check(level, Equals, release.FullAppArmor)
 	c.Check(summary, Equals, "apparmor is enabled and all features are available")
 }
+
+func (s *apparmorSuite) TestInterfaceSystemKey(c *C) {
+	fakeSysPath := c.MkDir()
+	restore := release.MockAppArmorFeaturesSysPath(fakeSysPath)
+	defer restore()
+	err := os.MkdirAll(filepath.Join(fakeSysPath, "policy"), 0755)
```
Comment (zyga) [inline release/apparmor_test.go]: Can you add "network" alongside policy please.

File: release/apparmor.go (hunk 1 above)
Comment (mvo5) [reply release/apparmor.go]: It is sorted, I added a comment.

File: release/apparmor_test.go (hunk 2 above)
Comment (mvo5) [reply release/apparmor_test.go]: Sure, done.

File: interfaces/system_key.go [hunk 3]
```diff
@@ -52,12 +49,7 @@ func generateSystemKey() *systemKey {
 	//
 	// We prefix the dirs.GlobalRootDir (which is usually "/") to make
 	// this testable.
-	if dentries, err := ioutil.ReadDir(filepath.Join(dirs.GlobalRootDir, "/sys/kernel/security/apparmor/features")); err == nil {
-		mySystemKey.AppArmorFeatures = make([]string, len(dentries))
-		for i, f := range dentries {
-			mySystemKey.AppArmorFeatures[i] = f.Name()
-		}
-	}
+	mySystemKey.AppArmorFeatures = release.AppArmorFeatures()
```
Comment (stolowski) [inline interfaces/system_key.go]: Nice.

**Pull request merged**